src/openforms/submissions/api/viewsets.py

```diff
@@ -23,6 +23,7 @@
 from openforms.api.filters import PermissionFilterMixin
 from openforms.api.serializers import ExceptionSerializer, ValidationErrorSerializer
 from openforms.api.throttle_classes import PollingRateThrottle
+from openforms.authentication.utils import is_authenticated_with_plugin
```

Comment:
please expose and import cross-application public API only through the `service.py` module

Reply:
oups :see_no_evil: